CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
   - Fix honoring of `includeCurrentStateVariables` flag in `WorkflowInstanceService.listWorkflowInstances`. This caused major slowness when using bulk workflows.
     To preserve the existing (incorrect) default behaviour in backwards compatible way the default value in `QueryWorkflowInstances.Builder` is changed to `true`. The REST API is unaffected.
     Especially in workflows with many children that use the `StateExecution.getAllChildWorkflows` method the performance impact can be high. Before 7.0.0 release, it is recommended to use `StateExecution.queryChildWorkflows(new QueryWorkflowInstances.Builder().setIncludeCurrentStateVariables(false).build())` if state variables are not needed.
+  - Improve child workflow final state execution speed by caching the parent workflow type
+  - Improve workflow execution type by not scanning for child workflows. If workflow wants to access child workflows they should fetch them using the executor getChildWorklfow methods. Not backwards compatible.
+  - Fix bulk forkflow concurrency limit calculation.
```

Comment:
```suggestion
  - Fix bulk workflow concurrency limit calculation.
```

Reply:
Was there anything to fix here actually? Could not find that change anymore.

Reply:
My changes were harmful -> removed :)